.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Fixes: #
+
+## Summary
+- Summary of the solution to fix the issue.
+- Include details of design or implementation choices made, tradeoffs considered, etc.
+- Link to design document (when applicable)
+
+## Type of Changes
+- [ ] New Feature
+- [ ] Bug-fix
+- [ ] Enhancement
```

Comment:
Looks ok to me. Enhancement feels a little open-ended and i am afraid everyone will start using this category for everything.

I would also like to think why are we creating this template. I personally think this data should help the reviewers with PR review prioritisation. Any other data/info we need here that will help with prioritisation?